### ISSUE TITLE
Gardian Node - Not completing sync ad stuck on Starting Node

### ISSUE THREAD

**Issue by simongoat** [thetatoken/guardian-mainnet-guide#68]
I've tried lots time to install the GN. Have got to 99.96 and to 100% sync. I've deleted datebase and re-booted several times. Now its stuck on 'starting Node' and doesn't go to syncing. I've uninstalled and re-installed the GN.

**Comment by Mendez849** [comment]
@simongoat Hello Mate, Sorry to hear about your issue, you can contact Theta Network Support on Telegram via the weblink www.t.me/thethetanetwork. They'll be happy to resolve your issue. Cheers.

**Comment by saintscribe69** [comment]
hi, did you manage to get this sorted? im having the same p[roblem

**Comment by simongoat** [comment]
No, not at all. I've stopped for a bit as it was getting too frustrating and taking up too much time. I went on Theta's discord but  there was not any more help there.
